packages/zksync-dai-bridge/.prettierrc

```diff
@@ -0,0 +1,20 @@
+{
+  "printWidth": 120,
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "all",
+  "proseWrap": "always",
+  "overrides": [
+    {
+      "files": "*.sol",
+      "options": {
+        "printWidth": 100,
+        "tabWidth": 2,
```

Comment:
```suggestion
        "tabWidth": 4,
```

Reply:
We should actually change it in every package and and get all the files modified, but we should do that in another PR. Let's for now just do it completely here.